app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
+
+
+class NotWearingMaskError(Exception):
+    pass
```

Comment:
You should consider adding descriptive error messages to your exceptions. This will help you to understand the context of the error when it is raised. You can achieve this by overriding the '__init__' method in your exception classes and passing the error message to the base Exception class.